code/modules/mob/living/carbon/xenomorph/mutators/strains/carrier/eggsac.dm

```diff
@@ -40,8 +43,44 @@
 	carrier.update_eggsac_overlays()
 	carrier.eggs_max = 12
 	carrier.egg_planting_range = 2
+	apply_behavior_holder(carrier)
 	return TRUE
 
+/datum/behavior_delegate/carrier_eggsac
+	name = "Eggsac Carrier Behavior Delegate"
+	var/list/eggs_sustained = list()
+	var/egg_sustain_cap = EGGSAC_OFF_WEED_EGGCAP
+	var/sustain_distance = 14
+
+/datum/behavior_delegate/carrier_eggsac/append_to_stat()
+	. = "Eggs sustained: [length(eggs_sustained)] / [egg_sustain_cap]"
```

Comment:
are you sure it's = instead of +=?

Reply:
I don't think it actually matters in this case. But comparing it to the same functionality on other behavior delegates I should be consistent with them, also returning it as a list.

https://github.com/Birdtalon/cmss13/blob/c7e8ff476a2fbae3654794173d38f941a2fcc775/code/modules/mob/living/carbon/xenomorph/XenoProcs.dm#L88-L90